### DISCUSSION TITLE
Domoticz web interface crashes after installing plugin

### DISCUSSION THREAD

**Discussion by StephanTrenn** [addiejanssen/domoticz-solaredge-modbustcp-plugin#26] [Q&A] [answered]
I just installed domoticz from scratch and tried to add my SolarEdge converter with the help of your plugin. Initially it seemed to work, I saw the plugin in the hardware drop-down list and entered the corresponding data. Once I added the hardware, I was however not able to see the devices and the web-interface became unresponsive. After removing the plugin the web-interface worked fine again. Some time ago I was able to get the data from my inverter into domoticz via a different method (can't remember the details now), so the problems seems not to be the inverter (SE3000H). I also tried to downgrade pymodbus 2.5.3 to 2.4.0 but this didn't resolve the problem. What should I do to find out what exactly the problem is?

**Comment by cdelaat** [comment]
There is an incompatibility with the newer version of python in the newest rasberry pi system named Bullseye.
https://www.domoticz.com/forum/viewtopic.php?f=6&t=37376&start=40
and
https://www.domoticz.com/forum/viewtopic.php?p=282496&sid=fb1225de4b42747f27a85ed292e2b6b0#p282496
I solved it by using the previous system Buster. Also I compiled domoticz from source under Bullseye and that works too.

**Comment by StephanTrenn** [comment, marked as answer]
I just wanted to report that with the latest Domoticz release (2022.2) the plugin works again without any apparent problems.

**Reply by addiejanssen** [reply]
Thanks for the feedback @StephanT80
